### DISCUSSION TITLE
is it possible to read replica only for selected queries

### DISCUSSION THREAD

**Discussion by Kasuhikdas** [sequelize/sequelize#16154] [Help & Questions] [answered]
Is it possible to use master as primary read and use read replica for selected queries only?
Got to know useMaster uses master for read query. But is it possible to make it default and use useMaster: false for selected queries.

**Comment by ephys** [comment, marked as answer]
You can configure default options for `sequelize.query` by using the [`query` constructor option](https://sequelize.org/api/v7/interfaces/_sequelize_core.index.options#query), which includes `useMaster`, which you could default to true:

```ts
new Sequelize({
  query: {
    useMaster: true,
  },
})
```

**Reply by Kasuhikdas** [reply]
got it. But how to use replica for selected query?

**Reply by ephys** [reply]
Set `useMaster` to false on individual method calls

**Reply by Kasuhikdas** [reply]
thanks it worked.
